docs/man_pages/project/testing/debug-android.md

```diff
@@ -1,6 +1,9 @@
 debug android
 ==========
 
+All you need to do to start debugging your app is to execute `tns debug android`. The NativeScript CLI will build, deploy, start your app, start Chrome DevTools and attach the debugger.
```

Comment:
This should be under the `Usage|Synopsis` part of the help (here -> https://github.com/NativeScript/nativescript-cli/pull/1801/files#diff-e7b2eaa70b7b2394cb03e78ab260308aR18 ). We print the table of usage first in all of the help content.